Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -894,7 +894,7 @@ public async Task ApplySorting( params DataGridSortColumnInfo[] columns )
 
         if ( SortMode == DataGridSortMode.Single )
         {
-            if ( columns.Length >= 1 )
+            if ( !columns.IsNullOrEmpty() )

```

Comment:
Isn't this different now? you'll never get to the `Count==1` You still need to check the Length **>=1**

Reply:
Nvm, it's a different collection, `SortByColumns` haha. Yea nice catch. :)

Reply:
Double checked the `IsNullOrEmpty` implementation:

```csharp
    public static bool IsNullOrEmpty<T>( this IEnumerable<T> collection )
    {
        return collection is null || !collection.Any();
    }
``` 

The predicate `!columns.IsNullOrEmpty()` will be true if the column has at least one item.

Reply:
Yes yes I gotcha, I miss-read the `elseif` I tought it was accessing the same collection.